app/models/pydantic/asset_metadata.py

```diff
@@ -75,6 +75,12 @@ class RasterTileSetMetadata(AssetBase):
     resolution: Optional[int]
 
 
+class COGMetadata(AssetBase):
+    block_size: int
+    srid: int
+    resampling: int
```

Comment:
Should these be 
block_size: TileBlockSize
srid: Srid
resampling: ResamplingMethod